### ISSUE TITLE
Can't look for lyrics when vpn is enabled

### ISSUE THREAD

**Issue by EmpireMatrix** [shub39/Rush#50]
I just can't look for any lyrics when my VPN is on. It just won't load. Otherwise, everything works perfectly.

**Comment by shub39** [comment]
Yeah the api doesn't like when vpn is used for some reason. I'll see what can be done, no guarantees

**Comment by EmpireMatrix** [comment]
Thanks a lot ! No stress, I just spotted that, so I wanted to share it so that you're aware of it.
